frank-doc-doclet/src/main/java/org/frankframework/frankdoc/Utils.java

```diff
@@ -307,22 +307,26 @@ private static String getLinkReplacement(String linkBody) {
 	public static String substituteJavadocTags(String text, FrankClass context) throws FrankDocException {
 		// Order matters here. {@literal}, {@value} and {@link} can be used inside {@code ...} blocks.
 		String step = replaceLiteralValue(text);
-		step = replaceClassFieldValue(step, context);
-		return replaceClassCodeValue(step);
+		step = replaceFieldValue(step, context);
+		return replaceCodeValue(step);
 	}
 
 	private static String replaceLiteralValue(String text) throws FrankDocException {
 		return replacePattern(text, JAVADOC_LITERAL_START_DELIMITER, StringEscapeUtils::escapeHtml4);
 	}
 
-	private static String replaceClassCodeValue(String text) throws FrankDocException {
-		return replacePattern(text, JAVADOC_CODE_START_DELIMITER, StringEscapeUtils::escapeHtml4);
+	private static String replaceCodeValue(String text) throws FrankDocException {
+		return replacePattern(text, JAVADOC_CODE_START_DELIMITER, Utils::getCodeValueReplacement);
 	}
 
-	private static String replaceClassFieldValue(String text, FrankClass context) throws FrankDocException {
+	private static String replaceFieldValue(String text, FrankClass context) throws FrankDocException {
 		return replacePattern(text, JAVADOC_VALUE_START_DELIMITER, s -> getClassFieldValueReplacement(s, context));
 	}
 
+	private static String getCodeValueReplacement(String codeBlock) {
```

Comment:
Misschien mis ik hier iets, maar dit zorgt toch weer voor andere output? Oude situatie was bv de platte string 'true', dat wordt nu '<code>true</code>', is dat dan wel wenselijk?

Reply:
Ok, helemaal goed, ik doelde vooral op de plaatjes van Niels hierboven